ember-flight-icons/tests/dummy/app/templates/design.hbs

```diff
@@ -1,441 +1,683 @@
+        <h4 class="text-xl font-medium text-gray-900">Generic</h4>
+        <p class="text-sm text-gray-500">
+          Glyph with standard outline
+          <ul>
+            <li>Preferred default style</li>
+          </ul>
         </p>
       </div>
       <div class="sm:flex-shrink-0">
         <div class="flow-root">
-          <img
-            class="w-32 mx-auto"
-            src="assets/images/icon-types-generic.png"
-            alt=""
-            role="none"
-          />
+          <img class="w-64" src="assets/images/icon-types-generic.png" alt="" role="none" />
         </div>
       </div>
     </div>
-    <div class="text-center sm:flex sm:text-left lg:block lg:text-center">
-      <div class="mt-3 sm:mt-0 sm:ml-6 lg:mt-6 lg:ml-0">
-        <h4 class="text-xl font-medium text-gray-900">
-          Filled
-        </h4>
-        <p class="mt-2 text-sm text-gray-500">
-          Glyph with a solid fill; use to show a toggled state or create contrast.
+    <div class="text-left sm:flex lg:block">
+      <div class="mt-3 sm:flex-auto sm:mt-0 sm:mr-6 lg:mt-6 lg:mr-0">
+        <h4 class="text-xl font-medium text-gray-900">Filled</h4>
+        <p class="text-sm text-gray-500">
+          Glyph with a solid fill
+          <ul>
+            <li>Toggled state or for contrast</li>
+          </ul>
         </p>
       </div>
       <div class="sm:flex-shrink-0">
         <div class="flow-root">
-          <img
-            class="w-32 mx-auto"
-            src="assets/images/icon-types-filled.png"
-            alt=""
-            role="none"
-          />
+          <img class="w-64" src="assets/images/icon-types-filled.png" alt="" role="none" />
         </div>
       </div>
     </div>
-    <div class="text-center sm:flex sm:text-left lg:block lg:text-center">
-      <div class="mt-3 sm:mt-0 sm:ml-6 lg:mt-6 lg:ml-0">
-        <h4 class="text-lg font-medium text-gray-900">
-          Off
-        </h4>
-        <p class="mt-2 text-sm text-gray-500">
-          Glyph with a strike-through; used to indicate a disabled state.
+    <div class="text-left sm:flex lg:block">
+      <div class="mt-3 sm:flex-auto sm:mt-0 sm:mr-6 lg:mt-6 lg:mr-0">
+        <h4 class="text-xl font-medium text-gray-900">Off</h4>
+        <p class="text-sm text-gray-500">
+          Glyph with a strike-through
+          <ul>
+            <li>Disabled state</li>
+          </ul>
         </p>
       </div>
       <div class="sm:flex-shrink-0">
         <div class="flow-root">
-          <img
-            class="w-32 mx-auto"
-            src="assets/images/icon-types-st.png"
-            alt=""
-            role="none"
-          />
+            <img class="w-28" src="assets/images/icon-types-st.png" alt="" role="none" />
+        </div>
+      </div>
+    </div>
+    <div class="text-left sm:flex lg:block">
+      <div class="mt-3 sm:flex-auto sm:mt-0 sm:mr-6 lg:mt-6 lg:mr-0">
+        <h4 class="text-xl font-medium text-gray-900">Contained</h4>
```

Comment:
question: Can we use the classes we've already defined?
```suggestion
        <h4 class="ds-h4">Contained</h4>
```

Reply:
ok lets adjust that one in a separate PR. 👍

Reply:
I already took care of this one, just need to fix the linting issues.